pylint_odoo/checkers/no_modules.py

```diff
@@ -243,6 +248,11 @@
     '__init__', 'setUp', 'tearDown',
 ]
 
+DFTL_DEPRECATED_FIELD_PARAMETERS = {
+    'digits_compute': 'digits',
+    'select': 'index',
+}
+
 
 class NoModuleChecker(BaseChecker):
```

Comment:
I didn't find a way to parse the options as dictionaries so I refactored the variable a bit and am using a regex to check for the deprecated field params. Please take a look.